package.json

```diff
@@ -26,7 +26,8 @@
   },
   "dependencies": {
     "promise-pjs": "1.1.3",
-    "web-activities": "1.4.0"
+    "web-activities": "1.4.0",
+    "yarn": "^1.5.1"
```

Comment:
Why is it in deps? Can it go to `devDependencies` instead?

Reply:
Yes, Moved.

Reply:
In fact, this is not required at all. Removed.